drivers/hud_data_quality_report/app/models/hud_data_quality_report/generators/fy2022/question_two.rb

```diff
@@ -29,7 +29,7 @@ def run_question!
       metadata = {
         header_row: [
           'Data Element',
-          'Client Doesn’t Know/Refused',
+          'Client Doesn\’t Know/Refused',
```

Comment:
I think the quote didn't get changed here.

Reply:
I think you are right :-)